transaction_test.go

```diff
@@ -85,6 +85,98 @@ func startTransactionInvalidTraceContext(t *testing.T, traceContext apm.TraceCon
 	tx.Discard()
 }
 
+func TestContinuationStrategy(t *testing.T) {
+	testCases := map[string]struct {
+		traceContext     apm.TraceContext
+		strategy         string
+		expectNewTraceID bool
+		expectSpanLink   bool
+	}{
+		"restart": {
```

Comment:
Should we include an "es" tracestate entry for this test, to show that "restart" is indiscriminate?

Reply:
Great point! Added!